packages/arb-token-bridge-ui/src/util/orbitChainsList.ts

```diff
@@ -511,3 +511,17 @@ export function getOrbitChains(
 
   return [...mainnetChains, ...testnetChains]
 }
+
+export function getInboxAddressFromOrbitChainId(chainId: number) {
+  if (chainId === ChainId.StylusTestnetV2)
+    return '0xcdcf1f59f5d4a65a3c67e1341f8b85cba50e0a7c'
```

Comment:
You should be able to access it with `l2Networks[chainId]?.ethBridge.inbox`

Reply:
Updared